bolt/lib/Target/AArch64/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ endif()
 
```

Comment:
I believe on `LLVM_LINK_COMPONENTS` ([line 2](https://github.com/llvm/llvm-project/pull/127969/files#diff-7de26258d1fb79bd66dcfe3e237bf146d9c8d2aebab4289c0edfa697778ee431L2))  we need `MCDisassembler` as on AArch64 I get linking errors.